tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md

```diff
@@ -33,3 +33,13 @@ The expected failures in this file are from features in the owncloud/ocis repo.
 - [apiSpaces/copySpaces.feature:163](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/copySpaces.feature#L163)
 - [apiSpaces/copySpaces.feature:260](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/copySpaces.feature#L260)
 - [apiSpaces/copySpaces.feature:261](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/copySpaces.feature#L261)
+- [apiSpaces/moveSpaces.feature:161](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/moveSpaces.feature#L161)
+- [apiSpaces/moveSpaces.feature:162](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/moveSpaces.feature#L162)
+- [apiSpaces/moveSpaces.feature:181](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/moveSpaces.feature#L181)
+- [apiSpaces/moveSpaces.feature:182](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/moveSpaces.feature#L182)
+- [apiSpaces/moveSpaces.feature:183](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/moveSpaces.feature#L183)
+- [apiSpaces/moveSpaces.feature:184](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/moveSpaces.feature#L184)
+- [apiSpaces/moveSpaces.feature:185](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/moveSpaces.feature#L185)
+- [apiSpaces/moveSpaces.feature:186](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/moveSpaces.feature#L186)
+- [apiSpaces/moveSpaces.feature:189](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiSpaces/moveSpaces.feature#L189)
```

Comment:
Sorry for late review. tests failed not because I can't copy or move. I'd sudgest to use: 
```
And for user "Alice" folder "testshare" of the space "Shares Jail" should contain these files:
    | personal.txt |
```
instead of:
```
And for user "Alice" the space "Shares Jail" should not contain these entries:
      | /testshare/personal.txt |
```

And create separate tests to checking PROPFIND with Depth infinity. 
we can have a lot of working tests instead of waiting for a third-party issue to be fixed

@phil-davis @individual-it

Reply:
@ScharfViktor yes that can be done. I can create separate issue to fix it. shall I?

Reply:
It's going to be great. You can create a new PR to improve this and that: https://github.com/owncloud/ocis/issues/4087#issuecomment-1195237093